v2_test.go

```diff
@@ -73,14 +73,6 @@ var invalidParseV2Tests = []struct {
 		newBufioReader(append(append(SIGV2, PROXY, TCPv6), lengthV6Bytes...)),
 		ErrInvalidLength,
 	},
-	{
-		newBufioReader(append(append(append(SIGV2, PROXY, TCPv4), lengthV6Bytes...), fixtureIPv6Address...)),
-		ErrInvalidLength,
-	},
-	{
-		newBufioReader(append(append(append(SIGV2, PROXY, TCPv6), lengthV6Bytes...), fixtureIPv4Address...)),
-		ErrInvalidLength,
-	},
```

Comment:
Why did you remove this? And why didn't you had tests to (reproduce and) validate your scenario?

Reply:
Removed these because they're testing for the invalid length that is higher than expected, which is what the `>=` now allows.

Good point, I'll create some tests for my scenario. Stay tuned.